### ISSUE TITLE
]Link.List fails to report active links

### ISSUE THREAD

**Issue by aplteam** [Dyalog/link#58]
I have Link working: it saved fns/opr/classes in `#.foo` when edited, and it brings them in from the file system as well.

However, when I execute any of
```
]Linkl.List 
]Linkl.List # 
]Linkl.List #.foo
```
I always get “No active links”.

**Comment by mkromberg** [comment]
@aplteam, can you provide instructions to reproduce this issue? I am unable to do so.

**Comment by aplteam** [comment]
Well, I tried it in several ways, none of which worked.

For example in a CLAER WS:

```
      ]link.create #.Kai D:\Temp\LinkTest
Linked: #.Kai ←→ D:\Temp\LinkTest
      ]link.list
* Command Execution Failed: SYNTAX ERROR
```

**Comment by mkromberg** [comment]
Strange, always works for me :-)

Can you try it again, after 

]udebug on

?

**Comment by aplteam** [comment]
I did this as well of course but it does not help:

```
      ]udebug on
      ]link.list
SYNTAX ERROR
Run[20] r←(⊃ôpts)(⎕SE.Link⍎cmd)ârgs
```

At which point I gave up because I don't have neither `â` nor `ô` on my keyboard which makes it nasty to investigate further. But okay, this time I did. 

I had to jump over the line ` :Trap DEBUG↓0` to see this:

```
SYNTAX ERROR
List[17] lineages←{⍕¨{∪(⊃⍵).##,⍵}⍣≡2⊃⍵}¨U.GetInfos ⍬
      ≢U.GetInfos ⍬
0
```

**Comment by aplteam** [comment]
I should mention that this is Link  2.06 which came with the beta couple of days ago.

**Comment by mkromberg** [comment]
Thanks, I should have mentioned you also need to set ⎕SE.Link.DEBUG←1. We'll think about whether the user commands should do this automagically when ]udebug on.

Thanks, now we are able to investigate!

**Comment by abrudz** [comment]
@aplteam Can you still reproduce this?

**Comment by aplteam** [comment]
Yes. But why wouldn't I? Shall I update something?

**Comment by abrudz** [comment]
@aplteam  Should be fixed with fff2e6e01b0873b6caa351a8fbc35e0f6afc13ab
